Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3986,11 +3986,11 @@ LinearMediaPlayerEnabled:
   condition: ENABLE(LINEAR_MEDIA_PLAYER)
   defaultValue:
     WebKitLegacy:
-      default: false
+      default: true
```

Comment:
I think you should also update the `status` field. `unstable` + `default: true` doesn't really make sense. Probably should be `status: embedder`.